app-admin/qtpass/qtpass-1.3.1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_P="QtPass-${PV}"
+inherit desktop qmake-utils
+
+DESCRIPTION="multi-platform GUI for pass, the standard unix password manager"
+HOMEPAGE="https://qtpass.org/"
+SRC_URI="https://github.com/IJHack/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="test"
+
+RDEPEND="app-admin/pass
+	dev-qt/qtcore:5
+	dev-qt/qtgui:5[xcb]
+	dev-qt/qtnetwork:5
+	dev-qt/qtwidgets:5
+	net-misc/x11-ssh-askpass"
+DEPEND="${RDEPEND}
+	dev-qt/linguist-tools:5
```

Comment:
in EAPI-7 this would be in `BDEPEND` instead.

Reply:
Yes, moved to BDEPEND.